src/components/studies/app-design/AppDesign.tsx

```diff
@@ -279,11 +282,13 @@ const AppDesign: React.FunctionComponent<
   onUpdate,
   onSave,
   study,
-
   onError,
 }: AppDesignProps & StudyBuilderComponentProps) => {
   const handleError = useErrorHandler()
 
+  let query = useQuery()
+  const showError = query.get('from') === 'launch'
```

Comment:
I think you don't need to check for a particular value e.g. launch. You know that 'launch' is the only place where it gets set. So you just want to check if it has some value. It also guards you agains misspelling e.g. 'lunch' value being passed